### PR TITLE
Use testHardpoint command to test the hardpoints

### PR DESCRIPTION
**Pull request by pkubanek**
- Use testHardpoint command to tests hardpoints
- Use print method to record changes

### CODE REVIEW

File: MTM1M3Movements.py [hunk 1]
```diff
@@ -652,30 +622,57 @@ async def hardpointActuatorData(self, data):
...
-        with click.progressbar(
-            [-999999999, 999999999],
-            label=click.style("Stepping HP", fg="green"),
-            item_show_func=lambda a: str(a),
-            show_pos=True,
-            width=0,
-        ) as bar:
-            for step in bar:
-                await self.hardpoint_move(step)
+        self.printTest(f"Testing Hardpoint Actuator {self.hp}")
```
Comment (b1quint) [inline MTM1M3Movements.py:637]: Please, use `snake_case` for these methods.

File: MTM1M3Test.py [hunk 2]
```diff
@@ -212,13 +204,10 @@ async def _raising(self):
...
                     ) / mdur
-                    self.max_raising_rate = max(
-                        self.max_raising_rate, raising_rate
-                    )
+                    self.max_raising_rate = max(self.max_raising_rate, raising_rate)
                 last_raising_ims = ims
                 if not (
-                    self.m1m3.evt_detailedState.get().detailedState
-                    == raisingState
+                    self.m1m3.evt_detailedState.get().detailedState == raisingState
```
Comment (b1quint) [inline MTM1M3Test.py:210]: I believe we want to use `.aget()` instead of `.get()` to avoid race conditions.

File: MTM1M3Movements.py (hunk 1 above)
Comment (pkubanek) [reply MTM1M3Movements.py:637]: snake_case converting is planned, but it needs to happen in single change - I cannot change only this, as it is used in multiple locations. There will be separate commit for case changing.

File: MTM1M3Test.py (hunk 2 above)
Comment (pkubanek) [reply MTM1M3Test.py:210]: Which race conditions? I need the current detailed state, I don't want the code to wait if one isn't available. There isn't any race associated with get, it simply retrieve value stored - no other code can write to variable location, that's how async works.

**Pull request merged**